website/source/docs/drivers/docker.html.md

```diff
@@ -625,7 +625,8 @@ options](/docs/agent/configuration/client.html#options):
   which is the list of capabilities allowed by docker by default, as 
   [defined here](https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities).
   Allows the operator to control which capabilities can be obtained by 
-  tasks using `cap_add` and `cap_drop` options.
+  tasks using `cap_add` and `cap_drop` options. Supports the value `"ALL"` as a 
```

Comment:
How can I set the "cap_add" & "cap_drop" in the .nomad file?

Reply:
Which version of Nomad are you using? This will be available in 0.8.0, which hasn't been released yet.
If you are using your own development build of 0.8.0 and you find the docs unclear I suggest you write to [Nomad mailing list](https://groups.google.com/group/nomad-tool) or if you suspect a bug, submit a new issue. There's a much higher chance of getting a response that way :)